changelog/unreleased/features/2284--parquet-store-plugin.md

```diff
@@ -0,0 +1,6 @@
+A new parquet store plugin allows VAST to store its data as parquet files,
+increasing storage efficiency at the expense of higher deserialization costs.
+Storage requirement for the archive (excluding indices) is reduced by 50%
```

Comment:
I think the "excluding indices" part is a bit too complicated for users, which would be interested the overall numbers instead.